### PR TITLE
RUMM-1939 Document data storage regarding encryption

### PR DESCRIPTION
**Pull request by maxep**
### What and why?

Document cleartext data storage in cache directory.

### Review checklist

- [x] Make sure each commit and the PR mention the Issue number or JIRA reference

### CODE REVIEW

File: docs/log_collection.md [hunk 1]
```diff
@@ -214,7 +214,9 @@ The SDK changes its behavior according to the new value. For example, if the cur
 - if changed to `.granted`, the SDK will send all current and future data to Datadog;
 - if changed to `.notGranted`, the SDK will wipe all current data and will not collect any future data.
 
-    When writing your application, you can enable development logs. All internal messages in the SDK with a priority equal to or higher than the provided level are then logged to console logs.
+> Before uploading to Datadog, data is stored in cleartext in the cache directory (`Library/Caches`) of your application bundle.
```
Comment (ncreated) [inline docs/log_collection.md:217]: Any reason for using quotation? I think it should be just a plain text:
```suggestion
Before uploading to Datadog, data is stored in cleartext in the cache directory (`Library/Caches`) of your application bundle.
```

File: docs/trace_collection.md [hunk 2]
```diff
@@ -212,6 +212,8 @@ For example, if the current tracking consent is `.pending`:
 - If changed to `.granted`, the SDK sends all current and future data to Datadog.
 - If changed to `.notGranted`, the SDK wipes all current data and does not collect future data.
 
+> Before uploading to Datadog, data is stored in cleartext in the cache directory (`Library/Caches`) of your application bundle.
```
Comment (ncreated) [inline docs/trace_collection.md:215]:
```suggestion
Before uploading to Datadog, data is stored in cleartext in the cache directory (`Library/Caches`) of your application bundle.
```

File: docs/log_collection.md (hunk 1 above)
Comment (maxep) [reply docs/log_collection.md:217]: Not really, it was more about rendering it as a note.

File: docs/log_collection.md [hunk 3]
```diff
@@ -214,7 +214,9 @@ The SDK changes its behavior according to the new value. For example, if the cur
 - if changed to `.granted`, the SDK will send all current and future data to Datadog;
 - if changed to `.notGranted`, the SDK will wipe all current data and will not collect any future data.
 
-    When writing your application, you can enable development logs. All internal messages in the SDK with a priority equal to or higher than the provided level are then logged to console logs.
+Before uploading to Datadog, data is stored in cleartext in the cache directory (`Library/Caches`) of your application sandbox.
+
+When writing your application, you can enable development logs. All internal messages in the SDK with a priority equal to or higher than the provided level are then logged to console logs.
```
Comment (maycmlee) [inline docs/log_collection.md:219]:
```suggestion
When writing your application, enable development logs to log to console all internal messages in the SDK with a priority equal to or higher than the provided level.
```

**Review by maycmlee — COMMENTED** [review]
Left some feedback.

**Review by ncreated — APPROVED** [review]
👌

**Pull request merged**